bigtop-packages/src/charm/kafka/layer-kafka/README.md

```diff
@@ -159,6 +159,36 @@ The external kafka client should now be able to access Kafka by using
 `kafka-0:9092` as the broker.
 
 
+## Network Interfaces
+
+In some network environments, you may want to restrict kafka to
+listening for incoming connections on a specific network interface
+(for example, for security reasons). To do so, you may pass either a
+network interface name or a CIDR range specifying a subnet to the
+``network_interface`` configuration variable. For example:
+
+  juju set-config kafka network_interface=eth0
+
+or
+
+  juju set-config kafka network_interface=10.0.2.0/24
+
+Each kafka machine in your cluster will lookup the IP address of that
+network interface, or find the first network interface with an IP
+address in the specified subnet, and bind kafka to that address.
+
+If you make a mistake, and pass in an invalid name for a network
+interface, you may recover by passing the correct name to set-config,
+and then running "juju resolved" on each unit:
+
+  juju set-config kafka network_interface=eth0
+  juju resolved kafka/0
```

Comment:
As with the zk charm, instructions should say `resolved -r` to retry the failed hook.  Without `-r`, config-changed will not re-run.